test/src/edu/stanford/nlp/io/FileSystemTest.java

```diff
@@ -0,0 +1,69 @@
+package edu.stanford.nlp.io;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import static edu.stanford.nlp.io.FileSystem.*;
+
+/**
+ * Test the FileSystem class.
+ *
+ * @author Julius Willems
+ */
+public class FileSystemTest {
```

Comment:
Interesting class, but also tricky. I believe we should keep this as a separate commit and not squash in the future.